dotcom-rendering/src/web/components/CardHeadline.tsx

```diff
@@ -84,16 +84,16 @@ const labTextStyles = (size: SmallHeadlineSize) => {
 	}
 };
 
-const underlinedStyles = (size: SmallHeadlineSize) => {
+const underlinedStyles = (size: SmallHeadlineSize, colour: string) => {
 	function generateUnderlinedCss(baseSize: number) {
 		return css`
 			background-image: linear-gradient(
 				to bottom,
 				transparent,
 				transparent ${baseSize - 1}px,
-				rgba(199, 0, 0, 0.5)
+				${colour}
 			);
-			line-height: ${baseSize - 1}px;
+			line-height: ${baseSize}px;
```

Comment:
Any idea why we removed it in the first place, @oliverlloyd?

See https://github.com/guardian/dotcom-rendering/pull/1003/files#r811988546

Reply:
2019?! I have no idea